app/models.py

```diff
@@ -0,0 +1,42 @@
+from app import db
+from werkzeug.security import generate_password_hash, check_password_hash
+
+
+class User(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(64), index=True, unique=True)
+    email = db.Column(db.String(64), index=True, unique=True)
+    email_verified = db.Column(db.Boolean)
+    password_hash = db.Column(db.String(1024))
+
+    def set_password(self, password) -> None:
+        self.password_hash = generate_password_hash(password)
+
+    def check_password(self, password) -> bool:
+        return check_password_hash(self.password_hash, password)
+
+class House(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    invite_link = db.Column(db.String(64), index=True, unique=True)
```

Comment:
I think we can generate this using a JSON Web Token – basically we would encrypt a message from the server, stick it in a link and then decrypt it later.

Reply:
```suggestion
```

Reply:
But wouldn't be nice to have a link that users can share between them? In this case we'd have to store it in the database

Reply:
I think we can generate them cryptographically!